test/fixtures/family_members.yml

```
@@ -0,0 +1,39 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  ci: MyString
+  rol: MyString
```

rol -> role, si es lo que yo pienso que es
```suggestion
  role: MyString
```

---

ya no existe el archivo, creo